docs/content/error/orderBy/notarray.ngdoc

```diff
@@ -0,0 +1,53 @@
+@ngdoc error
+@name orderBy:notarray
+@fullName Not an array
+@description
+
+This error occurs when {@link ng.orderBy filter} is not used with an array:
+```html
+<input ng-model="search">
+<div ng-repeat="(key, value) in myObj | orderBy:search">
+  {{ key }} : {{ value }}
+</div>
+```
+
+orderBy must be used with an array so a subset of items can be returned.
+The array can be initialized asynchronously and therefore null or undefined won't throw this error.
+
+To orderBy an object by the value of its properties you can create your own array based on that object:
+```js
+angular.module('aModule',[])
+  .controller('aController', function($scope) {
+    var obj = {
+      one:   {
+              name: 'something 1'
+              },
+      two:   {
+              name: 'something 2'
+             },
+      three: {
+             name: 'something 3'
+             },
```

Comment:
Also, the indentation is funny for this object. TBH, I would change the example to something like:

```html
<!-- HTML -->
<label>
  Order by:
  <select ng-model="orderProp" ng-options="prop for prop in ['id', 'name']"></select>
</label>
<div ng-repeat="(key, value) in myObj | orderBy:orderProp">
  {{ key }} : {{ value }}
</div>
```

```js
/* JS */
angular.module('aModule', [])
  .controller('aController', function($scope) {
    var myObj = {
      one: {id: 1, name: 'Some thing'},
      two: {id: 2, name: 'Another thing'},
      three: {id: 3, name: 'A third thing'}
    };

    $scope.arrFromMyObj = Object.keys(myObj).map(function(key) {
      return myObj[key];
    });
  });
```